### ISSUE TITLE
Stake Pools UI defaults to 1000 ADA on initial page load/view

### ISSUE THREAD

**Issue by mreed911** [input-output-hk/daedalus#2498]
When going to Stake Pools, the UI slider defaults to 1000 ADA.  Changing it causes stake pools to recalculate.

This should default to something other than 1000 and could be:

- The total ADA represented in all wallets in Daedalus (likely the best/most compatible option)
- The largest total in a single wallet in Daedalus
- The largest total in an unstaked wallet in Daedalus

This would allow me, as a user, to see the most relevant stake pool to my balance to be staked.

**Comment by nikolaglumac** [comment]
cc @darko-mijic

**Comment by srdjan** [comment]
also, after upgrading to the latest (4.0.5) no matter what amount I select using the slider - after stepping through tree step wizard, amount is fixed to 2 ADA?

<img width="337" alt="Screen Shot 2021-04-15 at 12 30 57 PM" src="https://user-images.githubusercontent.com/61190/114905354-dd8d5c80-9de6-11eb-947d-78c0c02bfe47.png">

**Comment by mreed911** [comment]
> also, after upgrading to the latest (4.0.5) no matter what amount I select using the slider - after stepping through tree step wizard, amount is fixed to 2 ADA?

Yes, that's the deposit to stake (plus the fee for the transaction), not the amount that's being staked (which is snapshotted in the wallet).  In your screenshot this is working correctly and isn't related to the problem I reported.

**Comment by srdjan** [comment]
agree - not related - deleted...

hope I'm the only one confused with this.

**Comment by mreed911** [comment]
> agree - not related - deleted...
> 
> hope I'm the only one confused with this.

Instead of replying - again - and cluttering the thread, why not delete THIS one, too, so we can keep things clean?

**Comment by nikolaglumac** [comment]
Thanks for the suggestion @mreed911 - we are discussing the options and are leaning towards having the default to be "The total ada represented in all wallets in Daedalus".

However, we also need to think of a case where users have no wallets or the total ada in their wallets is zero or less than 10 (which is the minimum slider value). What I am trying to say is that we need a default value for that case too and I believe that one should be 1000 ADA as it is now. Do you agree or do you have some other idea?

Thanks for helping out!

**Comment by mreed911** [comment]
Given the current price in USD of ADA, 1000 ADA represents a pretty significant investment vs. what someone just wading in might buy, especially as Alonzo rolls out and people can begin buying ADA to actually use, etc.

I'd suggest that "total ADA in all wallets" is a start point as long as that is > 10, and all the way down to 10, otherwise default to 10 (the minimum) if they have 10 or less, no wallets, etc. (just using the tool to see what their options are).

**Comment by disassembler** [comment]
Closing as part of issue backlog cleanup — issues older than 90 days are being retired to give the new team a clean slate. Please open a new issue if this is still relevant.
